### PR TITLE
Created education portfolio (rough draft)

### PR DESCRIPTION
**Pull request by kavixiu**
Here's the first draft of the education portfolio (visuals to come). Please add/change/edit! All feedback welcome. I'm wondering - is the word 'portfolio' confusing, since we are called Services Portfolio? Would 'gallery' or another word be better to show examples of our previous work?

### CODE REVIEW

File: course_development_kit/3_EducationPortfolio.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+When requesting educational materials, we ask that you review these examples of offerings to get an idea of what we are able to offer before making your request. This should give you a taste of what is available and help you to narrow down exactly what you are in need of, as well as an approximate timeframe of when you can expect a final product. Keep in mind that, depending on the scope of the project, turnaround times to reach a final product can range from weeks to months - it really comes down to what you want, and your particular project’s level of complexity and all the moving parts involved. These time estimates are intended more to give you an idea of the complexity involved, rather than a set date on which to expect a final product. 
+
+## Portfolio
+
+**Single-page, written course with screenshots**  
+Turnaround time: Approximately 2-6 weeks (small scope)  
+Details: Subject-matter experts are asked to write a rough draft of the learning content and to collect screen captures associated with their content. This first draft is then edited and reworked into a more polished form, and then pushed to Learndot via Github.   
+Example: Tasks course (written portion)
```
Comment (greglarkin) [inline course_development_kit/3_EducationPortfolio.md]: @kavixiu Can we link directly to the example content, or would you prefer to include screenshots only to replace the "visuals here" tags?

File: course_development_kit/3_EducationPortfolio.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Turnaround time: Depends on scope; approximately 4-8 weeks per video (medium scope)  
+Details: Subject-matter experts are asked to record a demo walkthrough using separate audio and video files and strict recording parameters using the highest quality settings. Video and audio is then synced, edited and reworked into its final form, before being pushed to Wistia. When complete, the video may be available via Learndot within a course, or may be hosted on Youtube as a standalone video or as part of a playlist.  
+Example: Puppet Learning VM Walkthrough video series (Youtube)
+<VISUALS HERE>
+
+**Training video with graphics and/or animations**  
+Turnaround time: Approximately 8-12 weeks (large scope)  
+Details: All educational content is contained within videos (course contains no written educational content). Audio and video content is developed in-house, and may consist of a combination of screenshots, illustrations, animations or captured footage. Due to the visual-design workload of these types of offerings, they will require a longer turnaround time. Final videos are hosted on Learndot (via Wistia) or Youtube.  
+Example: Beginner’s guide to Puppet and Devops
```
Comment (greglarkin) [inline course_development_kit/3_EducationPortfolio.md]: @kavixiu Is this example training video available on Learndot?

File: course_development_kit/3_EducationPortfolio.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+Turnaround time: Approximately 8-12 weeks (large scope)  
+Details: All educational content is contained within videos (course contains no written educational content). Audio and video content is developed in-house, and may consist of a combination of screenshots, illustrations, animations or captured footage. Due to the visual-design workload of these types of offerings, they will require a longer turnaround time. Final videos are hosted on Learndot (via Wistia) or Youtube.  
+Example: Beginner’s guide to Puppet and Devops  
+<VISUALS HERE>
+
+**Combined offerings (any combination of interactive, written, video, audio, etc.)**  
+Turnaround time: 12 weeks to 6 months+ (advanced scope)  
+Details: This offering is the most robust option and as a result takes the longest to develop. This offering is only available as a course hosted on Learndot.   
+Example: Tasks and Bolt course (Written content with screenshots; demo walkthrough video); Puppet Language Basics (Written content, illustrations, animations, video and interactive command-line tool); Value of Puppet for PSEs (Series of training videos contained within in a Learndot course)
```
Comment (greglarkin) [inline course_development_kit/3_EducationPortfolio.md]: @kavixiu Suggest changing course name to "Introduction to Bolt and Puppet Tasks"

File: course_development_kit/3_EducationPortfolio.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+## This document is intended for:
+
+* Shareholders who are requesting training materials from the Services Portfolio team. 
+* Subject-matter experts who are being requested to provide educational content for the Services Portfolio team.
+
+**Note to shareholders:**
```
Comment (bridgetegan) [inline course_development_kit/3_EducationPortfolio.md]: I would change this to stakeholders instead of shareholders

**Comment by kavixiu** [comment]
@greglarkin @bridgetegan  I'm going to close out this PR because I want to redesign the portfolio with less explainer content and more visuals, and also relocate it to Confluence to be more easily shared with stakeholders.

**Pull request closed without merging**